site-src/references/implementations.md

```diff
@@ -10,17 +10,19 @@ Implementors of Gateway API are encouraged to update this document with status i
 - [Emissary-Ingress (Ambassador API Gateway)][2] (alpha)
 - [Gloo Edge 2.0][3] (work in progress)
 - [Google Kubernetes Engine][4] (public preview)
-- [Istio][5] (alpha)
-- [Kong][6] (work in progress)
-- [Traefik][7] (alpha)
+- [HAProxy Ingress](5) (alpha)
```

Comment:
```suggestion
- [HAProxy Ingress][5] (alpha)
```

Reply:
Oh sure, my bad, used to use the `[text](link)` syntax and didn't see the difference. Fixed now.

Reply:
No worries, always throws me off too, thanks for the quick update!